### ISSUE TITLE
Vesting - Building spec error

### ISSUE THREAD

**Issue by ansonla3** [w3f/kusama-genesis-spec#39]
When building the spec, an error has occurred.

```
./target/release/polkadot build-spec --chain customSpec.json --raw > customSpecRaw.json
2020-03-04 22:12:16 Building chain spec
Error: Service(Other("Error parsing spec file: missing field `vesting` at line 16809 column 7"))
```

**Comment by lsaether** [comment]
We will need to add a separate "vesting" field to the Template.json since it was abstracted out of the Balance pallet recently.

**Comment by lsaether** [comment]
Addressed in d473474c6af9ca0abb9de9e72b533c3454e3b0e8
